gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceManager.java

```diff
@@ -265,6 +264,9 @@ public static GobblinServiceManager create(GobblinServiceConfiguration serviceCo
    * @param <T>
    */
   public static <T> T getClass(Class<T> classToGet) {
+    if (GOBBLIN_SERVICE_GUICE_MODULE == null) {
+      throw new RuntimeException("getClass called without calling create method to initialize GobblinServiceGuiceModule");
```

Comment:
include `classToGet` in the message so another clue toward who called it prematurely (in addition to stack trace)